tests/runfiles/linux.run

```diff
@@ -644,7 +644,8 @@ tests = ['rsend_001_pos', 'rsend_002_pos', 'rsend_003_pos', 'rsend_004_pos',
     'send-c_lz4_disabled', 'send-c_recv_lz4_disabled',
     'send-c_mixed_compression', 'send-c_stream_size_estimate', 'send-cD',
     'send-c_embedded_blocks', 'send-c_resume', 'send-cpL_varied_recsize',
-    'send-c_recv_dedup', 'send_encrypted_heirarchy', 'send_freeobjects']
+    'send-c_recv_dedup', 'send_encrypted_files', 'send_encrypted_heirarchy',
+    'send_freeobjects']
```

Comment:
A test case needs to be added to check for this errata.  You can create a tiny encrypted version 0 pool and include it with the ZTS for testing purposes.  The test case should.

* Verify version=0 filesystems be mounted read-only but not read-write.
* Verify version=0 volumes can opened read-only but not read-write.
* Verify the recommended `zfs send | zfs recv` conversion process works as intended.

Reply:
I was thinking about this. Would I just put the pool in the `zpool_import` test dir?

Reply:
Yes,  a similar thing is already done in the zpool import tests.